integration/mediation-tests/tests-other/src/test/java/org/wso2/carbon/esb/ServerStartupTestCase.java

```diff
@@ -17,8 +17,31 @@
  */
 package org.wso2.carbon.esb;
 
-import org.wso2.carbon.integration.common.tests.ServerStartupBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.annotations.ExecutionEnvironment;
+import org.wso2.carbon.automation.engine.annotations.SetEnvironment;
+import org.wso2.esb.integration.common.utils.CarbonLogReader;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 
-public class ServerStartupTestCase extends ServerStartupBaseTest {
+public class ServerStartupTestCase extends ESBIntegrationTest {
+
+    private CarbonLogReader carbonLogReader;
+
+    @BeforeSuite(alwaysRun = true)
+    public void initialize() {
+        carbonLogReader = new CarbonLogReader();
+    }
+
+    @Test(groups = { "wso2.esb" }, description = "verify server startup errors")
+    @SetEnvironment(executionEnvironments = { ExecutionEnvironment.STANDALONE })
+    public void testVerifyLogs() {
+        carbonLogReader.start();
+        String logs = carbonLogReader.getLogs();
+        boolean status = logs.indexOf("Mgt Console URL") < logs.indexOf("Starting WSO2 Carbon");
+        carbonLogReader.stop();
+        Assert.assertFalse(status, "Server started with errors.");
```

Comment:
Shouldn't we search for exception stack traces here? Having `Mgt Console URL` and `Starting WSO2 Carbon` will not be enough to say that the server started without errors. Don't you think?

Reply:
Done in #575